analyses/pypi/generatepylist.go

```diff
@@ -0,0 +1,16 @@
+package pypi
+
+// CodeForPylist is converted to generate_pylist.py
+var CodeForPylist = []byte(
```

Comment:
This need not be exported.
```suggestion
var codeForPylist = []byte(
```

Reply:
ack.